packages/nns/src/governance.canister.ts

```diff
@@ -370,6 +372,24 @@ export class GovernanceCanister {
     });
   };
 
+  /**
+   * Changes auto-stake maturity for this Neuron. While on, auto-stake maturity will cause all the maturity generated by voting rewards to this neuron to be automatically staked and contribute to the voting power of the neuron.
+   *
+   * @param {neuronId: NeuronId; autoStake: boolean;} params
```

Comment:
Is this a new way we should follow to write params?

Reply:
don't think it's new, there are several comments such as this one in the code base. output is nice.